### DISCUSSION TITLE
This is the best plugin ever!

### DISCUSSION THREAD

**Discussion by ziontee113** [nvim-neotest/neotest#94] [General]
Hi @rcarriga , thank you and all neotest contributors so much for this plugin! I just tried it out and it's amazing!

The amount of details / functionalities is just crazy to me. I can even jump to targeted files in the `summary` window? :scream: 

As someone who's new to TDD and testing with plenary, this is such a game changer.

As someone who wants to create nvim plugin this is a gold mine to learn from. Thank you again for your hard work! :+1:

**Comment by rcarriga** [comment]
Thanks for the kind words, always delighted to hear it helps people so much! Also great to hear it's being used to encourage more NeoVim development :smile:
